deploy/helm/kubecf/values.yaml

```diff
@@ -212,6 +212,27 @@ services:
       start: 20000
       end: 20008
 
+settings:
+  router:
+    # tls sets up the public TLS for the router. The tls keys:
+    #   ca: the CA in the PEM format. Not required if the certificate is signed by a well-known authority.
+    #   crt: the certificate in the PEM format. Required.
+    #   key: the private key in the PEM format. Required.
+    tls: {}
+    # ca: |
```

Comment:
Please document that, (for OpenSSL reasons) the CA cert _must_ be self-signed.  An intermediate does _not_ work here. (I will mention tomorrow morning if not supplying it works.)

Reply:
Hmm, so, for an LE cert, _not_ supplying a CA works fine.